### PR TITLE
Enable CORS for local development (#23)

### PR DESCRIPTION
**Pull request by Fijo-Lurk**

### CODE REVIEW

**Review by mblomdahl — APPROVED** [review]
We should add a mechanism for not setting this CORS header when running in production.

**Comment by Fijo-Lurk** [comment]
> We should add a mechanism for not setting this CORS header when running in production.

**Comment by Fijo-Lurk** [comment]
[Enable CORS for local development](https://trello.com/c/nU49xSgj/45-enable-cors-for-local-development)

File: Server/src/app.module.ts [hunk 1]
```diff
@@ -5,11 +5,19 @@ import { AppService } from './app.service';
 import { LocationModule } from './location/location.module';
 import config from 'ormconfig';
 import { Location } from './location/entities/location.entity';
+import { ConfigModule } from '@nestjs/config';
+import configuration from 'config/configuration';
```
Comment (mblomdahl) [inline Server/src/app.module.ts:10]: See comment on import grouping in #26, same here.

File: Server/src/app.module.ts [hunk 2]
```diff
@@ -5,11 +5,19 @@ import { AppService } from './app.service';
...
 
 @Module({
   imports: [
     TypeOrmModule.forRoot(config),
     LocationModule,
+    ConfigModule.forRoot({
+      envFilePath: `${process.cwd()}/config/env/${process.env.NODE_ENV}.env`,
```
Comment (mblomdahl) [inline Server/src/app.module.ts:16]: There should be a README instruction on how to set up these .env files!

File: Server/src/app.module.ts (hunk 1 above)
Comment (Fijo-Lurk) [reply Server/src/app.module.ts:10]: This will be fix in PR #26

File: Server/src/app.module.ts (hunk 2 above)
Comment (Fijo-Lurk) [reply Server/src/app.module.ts:16]: added a new PR (#27) with setup guide in README and added new script @mblomdahl

**Pull request merged**